File: 截断句子.cpp
class Solution {
public:
    string truncateSentence(string s, int k) {
        int cnt = 0;
        for(int i = 0; i < s.length(); i++){
            if(s[i] == ' '){
                cnt++;
                if(cnt == k){
                    return s.substr(0, i);
                }
            }
        }
        return s;
    }
};

File: 截断句子.py
class Solution:
    def truncateSentence(self, s: str, k: int) -> str:
        cnt = 0;
        length = len(s);
        for i in range(1, length):
            if s[i] == ' ':
                cnt += 1;
                if cnt == k:
                    return s[:i];
        return s[:length+1];
